### DISCUSSION TITLE
How can we get only the predicted bounding_boxes from the Text detection model?

### DISCUSSION THREAD

**Discussion by akashlp27** [mindee/doctr#982] [Q&A]
Hey, quick question I was working of merging of 2 different pipelines say, docTR detection and 'deep-text' recognition, I'm stuck in getting the bounding boxes so that I can crop them and use another model for detection. Currently I'm using this piece of code for achieving it but I'm not sure what is going wrong.....!, but the cropped images aren't correct, do I need to do any post processing.?

`single_img_doc = DocumentFile.from_images(r"C:\Users\user\Downloads\image017.jpg")
`

`det_model = db_resnet50(pretrained=True)`

```
det_predictor = DetectionPredictor(PreProcessor((1024, 1024), batch_size=1, mean=(0.798, 0.785, 0.772), std=(0.264, 0.2749, 0.287)), det_model)
detection = det_predictor(single_img_doc)
ordered_detection = [boxes[DocumentBuilder._sort_boxes(boxes[:, :4])[0]] for boxes in detection]
for page in range(len(detection)):
  bucket_word = []
  for word in range(len(detection[page])):
    xmin, ymin, xmax, ymax, score = detection[page][word]
    dy,dx,d = single_img_doc[page].shape
    xmin_ = int(xmin*dx)
    xmax_ = int(xmax*dx)
    ymin_ = int(ymin*dy)
    ymax_ = int(ymax*dy)
    word_img =  single_img_doc[page][ymin_:ymax_, xmin_:xmax_] #where this is the cropped image
```

**Comment by felixdittrich92** [comment]
Hi @akashlp27 :wave:,

with the main branch i was not able to reproduce your behavior (crops looks ok) could you maybe provide a test file to reproduce ? :smiley:
